homeassistant/components/smartthings/sensor.py

```diff
@@ -588,3 +589,8 @@ def unit_of_measurement(self):
         if self.report_name == "power":
             return POWER_WATT
         return ENERGY_KILO_WATT_HOUR
+
+    @property
+    def last_reset(self) -> datetime | None:
```

Comment:
We should only return this when report name is energy, not power.

Also please include the comment in the code that it's an exception.

Reply:
Done.